components/_components.scss

```diff
@@ -0,0 +1,242 @@
+  display: flex;
+  //   justify-content: center;
+  #tweet-btn {
+    text-align: center;
+    border: transparent;
+    border-radius: 200px;
+    color: #fff;
+    font-size: 2rem;
+    padding: 1.4rem 8rem;
+    background-color: $color-main;
+    &:hover {
+      background-color: $color-main-dark;
+      transition: all 0.3s;
+      cursor: pointer;
+    }
+    @include respond(tab-land) {
+      display: none;
+    }
+  }
+}
+
+.profile-image {
+  width: 5rem;
+  height: 5rem;
+  border-radius: 50%;
+  margin-right: 1rem;
+
+  img {
+    height: 100%;
+    width: 100%;
+    border-radius: 50%;
+  }
+}
+
+.mail {
+  display: inline-block;
+  transform: translateY(-2px);
+  color: $color-grey;
+}
+
+#textarea {
+  border: none;
+  outline: none;
+  appearance: none;
+  resize: none;
+
+  &::placeholder {
+    font-size: 2.2rem;
+  }
+
+  &:focus::placeholder {
+    color: transparent;
+    transition: all 0.2s;
+  }
+
+  &[type="text"] {
+    font-size: 1.6rem;
+    font-family: $roboto;
+  }
+}
+
+.center-card {
+  display: grid;
+  grid-template-columns: auto 1fr;
+
+  .profile-image.card-image {
+    grid-row: 1/5;
+    grid-column: 1/2;
+  }
+
+  &-title {
+    display: grid;
+    grid-template-columns: auto auto 1fr auto;
+    align-items: center;
+    &-icon {
+      display: inline;
+      padding: 0.6rem 0.8rem;
+      font-size: 1.6rem;
+      border-radius: 50%;
+      &:hover {
+        color: $color-main;
+        background-color: $color-hover-lightblue;
+        cursor: pointer;
+        transition: all 0.3s;
+      }
+    }
+
+    img {
+      margin: 0rem 0.2rem;
+    }
+  }
+
+  &:hover {
+    background-color: $color-hover-light;
+    cursor: pointer;
+    transition: all 0.3s;
+  }
+
+  &-para {
+    margin-bottom: 1.2rem;
+  }
+
+  .card-images {
+    object-fit: cover;
+  }
+
+  .center-card-image {
+    border: none;
+    border-radius: 10px;
+    height: 60rem;
+
+    img {
+      max-width: 100%;
+      max-height: 100%;
+      object-fit: cover;
+      border-radius: 10px;
+    }
+    @include respond(phone-sm) {
+      height: 40rem;
+    }
+  }
+
+  &-images {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 15rem);
+    max-height: auto;
+    margin-bottom: 0.5rem;
+
+    & > * {
+      border: 1px solid white;
+    }
+    .image-1 {
+      grid-row: 1/3;
+      grid-column: 1/2;
+
+      img {
+        border-radius: 10px 0 0 10px;
+        max-width: 100%;
+        height: 100%;
+        object-fit: cover;
+      }
+    }
+    .image-2 {
+      img {
+        border-radius: 0px 10px 0px 0px;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+      }
+      grid-row: 1/2;
+      grid-column: 2/3;
+    }
+    .image-3 {
+      img {
+        border-radius: 0px 0 10px 0px;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+      }
+      grid-row: 2/3;
+      grid-column: 2/3;
+    }
+
+    @include respond(phone-sm) {
```

Comment:
Nice to show exactly what you are resizing / making it responsive for. Showing exactly it's for a phone. I like it.